ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-release-1.0.yaml

```diff
@@ -311,6 +325,90 @@ tests:
         requests:
           cpu: 300m
           memory: 500Mi
+- always_run: false
+  as: test-e2e-disconnected
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      CATSRC_IMG: registry.redhat.io/redhat/redhat-operator-index:v4.19
+      CATSRC_NAME: redhat-operators
+      MIRROR_OPERATORS: "true"
+    test:
+    - ref: cert-manager-install-catalog
+    - ref: kueue-operator-image-env-setup
+    - as: mirror-bundle
+      commands: |
+        source "${SHARED_DIR}/env"
+        echo "Using Bundle Image: ${BUNDLE_IMAGE}"
+        
+        # In disconnected environment, mirror the bundle image to the local registry
+        if [[ -f "${SHARED_DIR}/mirror_registry_url" ]]; then
+          echo "Disconnected environment detected - mirroring bundle image"
+          MIRROR_REGISTRY=$(cat "${SHARED_DIR}/mirror_registry_url")
+          echo "Mirror registry: ${MIRROR_REGISTRY}"
+          
+          # Extract bundle image name and tag
+          BUNDLE_IMAGE_NAME=$(echo "${BUNDLE_IMAGE}" | cut -d':' -f1)
+          BUNDLE_IMAGE_TAG=$(echo "${BUNDLE_IMAGE}" | cut -d':' -f2)
+          BUNDLE_IMAGE_BASENAME=$(basename "${BUNDLE_IMAGE_NAME}")
+          
+          # Create mirrored bundle image reference
+          MIRRORED_BUNDLE_IMAGE="${MIRROR_REGISTRY}/kueue-operator-tenant/${BUNDLE_IMAGE_BASENAME}:${BUNDLE_IMAGE_TAG}"
+          echo "Mirrored bundle image: ${MIRRORED_BUNDLE_IMAGE}"
+          
+          # Mirror the bundle image
+          echo "Mirroring bundle image to disconnected registry..."
+          oc image mirror "${BUNDLE_IMAGE}" "${MIRRORED_BUNDLE_IMAGE}" --insecure=true --skip-verification=true
+          
+          # Save the mirrored bundle image reference for the next step
+          echo "${MIRRORED_BUNDLE_IMAGE}" > "${SHARED_DIR}/mirrored_bundle_image"
+          echo "Saved mirrored bundle image reference: ${MIRRORED_BUNDLE_IMAGE}"
+        else
+          # Save the original bundle image reference for the next step
+          echo "${BUNDLE_IMAGE}" > "${SHARED_DIR}/mirrored_bundle_image"
+          echo "Using original bundle image: ${BUNDLE_IMAGE}"
+        fi
+      env:
+      - default: /shared
+        name: SHARED_DIR
+      from: cli
+      resources:
+        requests:
+          cpu: 300m
+          memory: 300Mi
+    - as: run-sdk
+      commands: |
+        source "${SHARED_DIR}/env"
+        
+        # Read the bundle image reference (original or mirrored)
+        BUNDLE_IMAGE=$(cat "${SHARED_DIR}/mirrored_bundle_image")
+        echo "Using bundle image: ${BUNDLE_IMAGE}"
+        
+        # Use operator-sdk from the operator-sdk base image
+        operator-sdk run bundle --timeout=15m -n openshift-kueue-operator --security-context-config restricted "$BUNDLE_IMAGE"
+      env:
+      - default: /shared
+        name: SHARED_DIR
```

Comment:
`SHARE_DIR` is a special ENV var reserved by build farm cluster, I do not think it is a good idea to overwrite it as a common step EVN var here.